exampleCourse/questions/elementDropdown/question.html

```diff
@@ -0,0 +1,20 @@
+	<p> Select the correct word in the following quotes:</p>
+	The 
+	<pl-dropdown sort="ascend" correct-answer="aristotle" blank="true">
+		<pl-answer correct="{{params.aristotle.tag1}}">{{params.aristotle.ans1}}</pl-answer>
```

Comment:
This doesn't show the usefulness of `server.py` by just taking some values defined elsewhere and copying them verbatim into here. Maybe `server.py` should randomly add some junk answers, and this file can use mustache loops in here to display each option, instead of listing them all explicitly.

Reply:
We need to list the answers here explicitly to fit the context of the sentences. Randomization will not work in this context.

Reply:
I will leave this code as is. I think it is fairly simple to randomize it further if desired by a question writer.